src/helpers/index.js

```diff
@@ -33,96 +33,53 @@ function parseOptions(options = {}, version = 3, green = false) {
 
   const adjustments = {};
 
-  if (options?.gridIntensity) {
-    adjustments.gridIntensity = {};
-    const { device, dataCenter, network } = options.gridIntensity;
-    if (device || device === 0) {
-      if (typeof device === "object") {
-        if (!averageIntensity.data[device.country?.toUpperCase()]) {
-          console.warn(
-            `"${device.country}" is not a valid country. Please use a valid 3 digit ISO 3166 country code. \nSee https://developers.thegreenwebfoundation.org/co2js/data/ for more information. \nFalling back to global average grid intensity.`
-          );
-          adjustments.gridIntensity["device"] = {
-            value: globalGridIntensity,
-          };
-        }
-        adjustments.gridIntensity["device"] = {
-          country: device.country,
-          value: parseFloat(
-            averageIntensity.data[device.country?.toUpperCase()]
-          ),
-        };
-      } else if (typeof device === "number") {
-        adjustments.gridIntensity["device"] = {
-          value: device,
-        };
-      } else {
-        adjustments.gridIntensity["device"] = {
-          value: globalGridIntensity,
-        };
-        console.warn(
-          `The device grid intensity must be a number or an object. You passed in a ${typeof device}. \nFalling back to global average grid intensity.`
-        );
-      }
-    }
-    if (dataCenter || dataCenter === 0) {
-      if (typeof dataCenter === "object") {
-        if (!averageIntensity.data[dataCenter.country?.toUpperCase()]) {
-          console.warn(
-            `"${dataCenter.country}" is not a valid country. Please use a valid 3 digit ISO 3166 country code. \nSee https://developers.thegreenwebfoundation.org/co2js/data/ for more information.  \nFalling back to global average grid intensity.`
-          );
-          adjustments.gridIntensity["dataCenter"] = {
-            value: SWDM3_GLOBAL_GRID_INTENSITY,
```

Comment:
Was this meant to remain as `SWDM3_GLOBAL_GRID_INTENSITY`? By moving this into the function it will use `globalGridIntensity` like the others.

Reply:
Thanks for catching this.